src/io/dsv.rs

```diff
@@ -121,7 +127,7 @@ impl DSVReader {
 
     /// Static function to create a serde reader
     ///
-    /// The function takes an arbitrary reader and wraps it into a serde-reader for csv
+    /// The function takes an arbitrary [`Reader`][Read] and wraps it into a [`serde-reader`][csv::Reader] for csv
```

Comment:
It's not actually a `serde-reader`, though – serde is only used for `csv::Reader::deserialize` and `csv::Reader::deserialize_into`, which we don't call. Why not just leave it at `csv::Reader`, which would be accurate _even if_ this had anything to do with serde?